common/djangoapps/student/tests/test_roles.py

```diff
@@ -142,6 +141,18 @@ def test_add_users_doesnt_add_duplicate_entry(self):
         role.remove_users(self.student)
         assert not role.has_user(self.student)
 
+    def test_get_org_for_user(self):
+        """
+        Test get_org_for_user
+        """
+        role = OrgContentCreatorRole(org=self.orgs[0])
+        assert len(list(role.get_org_for_user(self.student))) == 0
```

Comment:
nit: No need to call `list` here, since your `get_org_for_user` function already returns a list.